src/components/MerchantsPage/MerchantsPage.tsx

```diff
@@ -76,13 +78,17 @@ const MerchantsPage = (props: Props) => {
                 <h2 style={{ fontWeight: 'bolder' }}>Our Chinatown</h2>
                 <br />
                 <p>
-                  We are providing an online platform to low-tech, cash-only,
-                  Asian-owned small businesses that have been disproportionately
-                  impacted by COVID-19.
+                  {t(`
```

Comment:
This should have the key, and the value should be the string itself

Reply:
...that makes a lot more sense ahha, I was following what was done in the DonationPool component, but I guess that wasn't set up properly either. I refactored both components' text to `translation.json`